field_test.go

```diff
@@ -46,9 +47,19 @@ func assertFieldJSON(t testing.TB, expected string, field Field) {
 	enc := newJSONEncoder()
 	defer enc.Free()
 
+	var out interface{}
+	err := json.Unmarshal([]byte("{"+expected+"}"), &out)
+	require.NoError(t, err,
+		"Expected JSON snippet '%s' must be valid for use in an object.", expected)
+
 	field.AddTo(enc)
 	assert.Equal(t, expected, string(enc.bytes),
 		"Unexpected JSON output after applying field %+v.", field)
+	eb := append([]byte{'{'}, enc.bytes...)
```

Comment:
none that i can think of. 

i did this section first and then thought "i should also validate that what's passed in as expected is valid as well"